test/check-storage-mountpoints-btrfs

```diff
@@ -171,6 +179,18 @@ class TestStorageMountPointsBtrfs (VirtInstallMachineCase):
         """)
 
     def testEncryptedUnlockBTRFSonLUKS(self):
+        """
+        Description:
+            Test that the installer can handle BTRFS on LUKS encrypted devices
+
+        Expected results:
+            - The 'Unlock' dialog is displayed before entering the mount point mapping
+              screen.
```

Comment:
I'd rather say
`- The 'Unlock' dialog is available for unlocking the partitions before entering the mount point mapping`